src/net/java/sip/communicator/impl/protocol/jabber/ChatRoomMemberJabberImpl.java

```diff
@@ -32,6 +35,7 @@
  * @author Emil Ivov
  */
 public class ChatRoomMemberJabberImpl
+    extends DataObject
```

Comment:
Do we still need that? I don't see any of the `DataObject` methods being used.

Reply:
Yep, it is needed (we synced up offline with Nik and Boris)